### DISCUSSION TITLE
Metaphor Story - React Table - Sever Side Pagination, Search, Sort/Order | Metaphore (SCP)

### DISCUSSION THREAD

**Discussion by giscus** [StreetCommunityProgrammer/metaphore#66] [Metaphor - Discussion]
# Metaphor Story - React Table - Sever Side Pagination, Search, Sort/Order | Metaphore (SCP)

darkterminal telling a story about React Table - Sever Side Pagination, Search, Sort/Order

https://metaphore.vercel.app/stories/javascript/react-table-sever-side-pagination-search-sortorder

**Comment by ivanbilobrk** [comment]
Do you have CSS for this paginated table?

**Reply by darkterminal** [reply]
I use tailwindcss in my project btw...  :eyes:

**Reply by ivanbilobrk** [reply]
Thank you!
